### PR TITLE
Video Listing Page

### PR DESCRIPTION
**Pull request by arpit01923**
Description-
- videoListing page should be added
- contains a static data
- add components navbar, sidebar, main, card, footer

File to check-
- src/pages/video-listing/video-listing.jsx
- src/styles/Navbar.css
- src/styles/Footer.css
- src/pages/video-listing/main/video-card.jsx

Preview-
image

### CODE REVIEW

**Review by manojncsoni007 — APPROVED** [review]
looks good to me
- Nice UI
- Clean code

File: src/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { VideoListing } from "./video-listing/video-listing";
```
Comment (bajajcodes) [inline src/pages/index.js:1]: This can be done in single statement.

File: src/components/footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "../../styles/Footer.css";
+
+const Footer = () => (
+  <footer className="main-footer">
+    <h2 className="footer-header">CONTACT US</h2>
+
+    <div className="footer-link">
+      <a
+        className="link"
```
Comment (bajajcodes) [inline src/components/footer/Footer.jsx]: Make this repeated code compoenent using static data.

File: src/pages/video-listing/main/main.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { videos } from "../../../backend/db/videos";
```
Comment (bajajcodes) [inline src/pages/video-listing/main/main.jsx]: Can use absolute import.

File: src/pages/video-listing/video-listing.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { Footer } from "../../components/footer/Footer";
```
Comment (bajajcodes) [inline src/pages/video-listing/video-listing.jsx]: Can use absolute imports, to make imports shorter and easier.

File: src/styles/video-listing.css [hunk 5]
```diff
@@ -0,0 +1,113 @@
+:root{
+  --bg-color: #e2e8f0;
```
Comment (bajajcodes) [inline src/styles/video-listing.css]: Root values can be imported from one file.

**Review by bajajcodes — COMMENTED** [review]
Can merge, post changes.

File: src/pages/index.js (hunk 1 above)
Comment (arpit01923) [reply src/pages/index.js:1]: not working

Comment (bajajcodes) [reply src/pages/index.js:1]: How your implementing it?

**Pull request merged**